iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/ContentIntegrationTests.cpp

```diff
@@ -9737,6 +9925,76 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, LoadsPrimitiveArrayProperty
         << "Actual: \r\n" << BeRapidJsonUtilities::ToPrettyString(recordJson2["Values"]);
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsitest
++---------------+---------------+---------------+---------------+---------------+------*/
+DEFINE_SCHEMA(LoadsGuidArrayPropertyValue, R"*(
+    <ECEntityClass typeName="MyClass">
+        <ECArrayProperty propertyName="GuidsArrayProperty" typeName="binary" extendedTypeName="BeGuid" />
+    </ECEntityClass>
+)*");
+TEST_F(RulesDrivenECPresentationManagerContentTests, LoadsGuidArrayPropertyValue)

```

Comment:
I think we're missing a `LoadsGuidPropertyValue` test

Reply:
Added